tests/nodes/primary_nodes/test_project.py

```diff
@@ -95,3 +98,46 @@ def test_integration_project(cript_api, simple_project_node):
 
     # ========= test delete =========
     delete_integration_node_helper(cript_api=cript_api, node_to_delete=simple_project_node)
+
+
+@pytest.mark.skip(reason="api")
+def test_save_project_node(cript_api, simple_project_node, complex_project_node):
+    """
+    pytest nodes/primary_nodes/test_project.py::test_save_project_node
+    """
+
+    # with cript.API(host="https://lb-stage.mycriptapp.org/") as api:
+    #     with open("new_project.json") as json_handle:
+    #         proj_json = json.load(json_handle)
+
+    # proj = cript_api.load_nodes_from_json(nodes_json=proj_json)
+    # Modify deep in the tree
+    print("------------\nstarting")
+    proj0 = copy.deepcopy(complex_project_node)
+    proj_json = proj0.get_json().json
+    cript_api.save_new(proj0)
+
+    proj = cript.load_nodes_from_json(nodes_json=json.dumps(proj_json))
```

Comment:
also be careful here, if you load an existing project with the same UUID.
It will give you the same nodes as you have before.

You can't have different python nodes with the same UUID, except if you use the `_use_cache` argument of the load function.